docker/docker-compose.dev.yml

```diff
@@ -36,7 +39,7 @@ services:
       - EMAIL_SMTP_SENDER_EMAIL=${EMAIL_SMTP_SENDER_EMAIL}
     volumes:
       - "../:/usr/app"
-      - /usr/app/node_modules/ # clears the node_modules directory so it doesn't sync (v.slow on MacOS)
+      - "node_modules:/usr/app/node_modules/" # clears the node_modules directory so it doesn't sync (v.slow on MacOS)
```

Comment:
What difference does this make ?

Reply:
For me it wasn't persisting the `node_modules` between restarts and now it does!